packages/apps/src/app.process.ts

```diff
@@ -124,8 +124,8 @@ export async function $process<TPlugin extends IPlugin>(
   }
 
   const send = context.send.bind(context);
-  context.send = async (activity: ActivityLike) => {
-    const res = await send(activity);
+  context.send = async (activity: ActivityLike, conversationRef?: ConversationReference) => {
```

Comment:
why is this necessary when the activity context already has the conversation reference on it? https://github.com/microsoft/teams.ts/blob/3725bffd404756610c29cb969adfde66567f3bd1/packages/apps/src/contexts/activity.ts#L161